### ISSUE TITLE
Copy/pasting to new pathway

### ISSUE THREAD

**Issue by mkutmon** [PathVisio/pathvisio#12]
_Ticket submitted by Thomas:_

When copying something into a new pathway fails with a NullPointerException. 
Reproduce:
- Open/create a pathway
- Select all (CTRL-A)
- Copy (CTRL-C)
- Create a new pathway
- Paste (CTRL-V)

Expected behavior: the contents of the original pathway should appear in the new patwhay.

**Comment by mkutmon** [comment]
It seems to work when I click CTRL-A, then deselect the InfoBox (top left) and then paste it in a new pathway.

**Comment by DeniseSl22** [comment]
Yes, that is what I do as well; perhaps there cannot be two fields in the GPML for the infobox (title+species). Do we want people to have a "Select All" option, without the infobox (since there is no need to select that with all the other stuff in the GPML anyway?).
